scalr/resource_scalr_provider_configuration_test.go

```diff
@@ -444,7 +533,17 @@ func testAccCheckProviderConfigurationResourceDestroy(s *terraform.State) error
 	return nil
 }
 
-func testAccScalrPorivderConfigurationCustomConfig(name string) string {
+func getScalrTestingCreds(t *testing.T) (scalrHostname, scalrToken string) {
+	scalrHostname = os.Getenv("SCALR_HOSTNAME")
+	scalrToken = os.Getenv("SCALR_TOKEN")
+	if len(scalrHostname) == 0 ||
+		len(scalrToken) == 0 {
+		t.Skip("Please set SCALR_HOSTNAME, SCALR_TOKEN env variables to run this test.")
```

Comment:
`SCALR_HOSTNAME` and `SCALR_TOKEN` can be retrieved  from DefaultConfig/client and are mandatory for all tests. `getScalrTestingCreds` is not needed